src/bioregistry/data/bioregistry.json

```diff
@@ -240,6 +245,12 @@
     }
   },
   "aaindex": {
+    "contact": {
+      "email": "shuichi@hgc.jp",
+      "github": "skwsm",
+      "name": "Shuichi Kawashima",
+      "orcid": "0000-0001-7883-3756"
+    },
```

Comment:
This is a remaining example of `contact` added under a mapping key (in this case ``aaindex``). Generally, the contact info curated in bioregistry should be at the top level within the dictionary corresponding to the prefix.

Reply:
can you elaborate this a bit. I am having a bit of an issue understanding this.

Reply:
Sorry, I misread the structure, this is actually correct.